mesos-docker/run/run.sh

```diff
@@ -354,7 +355,7 @@ function start_slaves {
 
   for i in `seq 1 $NUMBER_OF_SLAVES`;
   do
-    start_slave_command="nohup /usr/local/bin/wrapdocker ; /usr/sbin/mesos-slave --master=$master --ip=$dip $(quote_if_non_empty $MESOS_SLAVE_CONFIG)"
+    start_slave_command="nohup /usr/local/bin/wrapdocker; /usr/sbin/mesos-slave --master=$master --work_dir=/tmp --ip=$dip $(quote_if_non_empty $MESOS_SLAVE_CONFIG)"
```

Comment:
Yes they do, but also kept those binaries. This code part is old not related to this pr. Maybe it would be good to update it in a new pr.